src/services/room-service.spec.js

```diff
@@ -276,6 +302,33 @@ describe('room-service', () => {
         const invitationFromDb = await db.roomInvitations.findOne({ _id: invitation._id });
         expect(invitationFromDb).toBeNull();
       });
+
+      describe('and the user gets invited a second time', () => {
+        let exisingMemberJoinDate;
+
+        beforeEach(async () => {
+          ({ invitation } = await sut.createOrUpdateInvitation({ roomId: testRoom._id, email: otherUser.email, user: myUser }));
+
+          const roomFromDb = await db.rooms.findOne({ _id: testRoom._id });
+          exisingMemberJoinDate = roomFromDb.members[0].joinedOn;
```

Comment:
Typo in here.

Also I find it simpler to not stray with namings in too many directions, makes code harder to follow.
I would suggest `existingMemberJoinedOn`.

Reply:
Fixed